### PR TITLE
Expose builtin entities ontology in Python wrapper

### PR DESCRIPTION
**Pull request by ratamovic**
…etrieval API

### CODE REVIEW

File: python/snips_nlu_parsers/builtin_entities.py [hunk 1]
```diff
@@ -196,3 +198,32 @@ def get_builtin_entity_examples(builtin_entity_kind, language):
...
+    global _LANGUAGE_ENTITY_ONTOLOGY_JSON
+    if language not in _LANGUAGE_ENTITY_ONTOLOGY_JSON:
+        with string_pointer(c_char_p()) as ptr:
+            exit_code = lib.snips_nlu_ontology_language_entity_ontology_json(
+                language.encode("utf8"), byref(ptr))
+            check_ffi_error(exit_code, "Something went wrong when retrieving "
+                                       "language entity ontology")
+            result = string_at(ptr)
+            _LANGUAGE_ENTITY_ONTOLOGY_JSON[language] = result.decode("utf8")
+    return _LANGUAGE_ENTITY_ONTOLOGY_JSON[language]
```
Comment (adrienball) [inline python/snips_nlu_parsers/builtin_entities.py]: As we do not really need the raw JSON string, the json loading should probably be done inside these two functions, which should be renamed without the `_json` suffix.

File: python/snips_nlu_parsers/tests/test_builtin_entities.py [hunk 2]
```diff
@@ -101,3 +103,20 @@ def test_should_get_builtin_entity_examples(self):
...
                 self.assertGreaterEqual(len(examples), 1)
+
+    def test_should_get_complete_entity_ontology_json(self):
+        # When
+        complete_ontology_json = json.loads(get_complete_entity_ontology_json(), encoding="utf8")
+
+        # Then
+        self.assertTrue(len(complete_ontology_json) > 0)
+        for configuration in complete_ontology_json:
+            self.assertTrue("language", "entities" in configuration)
```
Comment (adrienball) [inline python/snips_nlu_parsers/tests/test_builtin_entities.py]: It should rather be:
```python
self.assertTrue("language" in configuration)
self.assertTrue("entities" in configuration)
```

File: python/snips_nlu_parsers/builtin_entities.py (hunk 1 above)
Comment (ratamovic) [reply python/snips_nlu_parsers/builtin_entities.py]: k

File: python/snips_nlu_parsers/tests/test_builtin_entities.py (hunk 2 above)
Comment (ratamovic) [reply python/snips_nlu_parsers/tests/test_builtin_entities.py]: k

**Pull request merged**